connect/0.10/docker-debezium-download.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
@renatomefi Does it make sense to make a single library shell script file located in `connect-base` that would contain functions for obtaining deps from Maven central and confluent and alss function to download debezium connectors? Then other scripts will just source this library file un use the functions.

Reply:
@jpechane I think it makes sense.
I can also enforce the environment variables to be there